### PR TITLE
challenge-programs: add description for UCP season 2

### PR DESCRIPTION
**Pull request by dcalvin**
This PR added english version of readme for UCP season 2.

### CODE REVIEW

File: challenge-programs/challenge-program-season-2.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+1. Create a GitHub account. See [Join GitHub](https://github.com/join) for more information.
+2. [Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) in your local environment.
+3. Configure your Git to access Github. See [Set up Git](https://help.github.com/en/github/getting-started-with-github/set-up-git) for more information.
+
+## Sign up
+
+To sign up for the program, [Open an Issue](https://github.com/tidb-challenge-program/register) in the [tidb-challenge-program/register](https://github.com/tidb-challenge-program/register) Repo, in the following format:
+
+- Title: PCP/Sign Up
```
Comment (gingerkidney) [inline challenge-programs/challenge-program-season-2.md]:
```suggestion
- Title: UCP/Sign Up
```

File: challenge-programs/challenge-program-season-2.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+Of course, if you have other ideas of contributions that are not presented under the TODO column, you can raise a proposal by opening a new Issue to the corresponding Repo. Before you do that, make sure you have read the [Contribution Guide](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md) and [Contributor Covenant Code of Conduct](https://github.com/pingcap/community/blob/master/CODE_OF_CONDUCT.md).
+
+ - Raise a proposal to TiDB Repo：[https://github.com/pingcap/tidb/issues/new?labels=type%2Fenhancement&template=feature-request.md](https://github.com/pingcap/tidb/issues/new?labels=type%2Fenhancement&template=feature-request.md)
+ - Raise a proposal to TiKV Repo：[https://github.com/tikv/tikv/issues/new?template=feature-request.md](https://github.com/tikv/tikv/issues/new?template=feature-request.md)
+ - Raise a proposal to PD Repo：[https://github.com/pingcap/pd/issues/new?labels=type%2Fenhancement&template=feature-request.md](https://github.com/pingcap/pd/issues/new?labels=type%2Fenhancement&template=feature-request.md)
+
+**Format requirement**: Add the mark `REQ-PCP` at the beginning of the issue title, for example:
```
Comment (gingerkidney) [inline challenge-programs/challenge-program-season-2.md]:
```suggestion
**Format requirement**: Add the mark `REQ-UCP` at the beginning of the issue title, for example: 
```

File: challenge-programs/challenge-program-season-2.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+ - Raise a proposal to TiDB Repo：[https://github.com/pingcap/tidb/issues/new?labels=type%2Fenhancement&template=feature-request.md](https://github.com/pingcap/tidb/issues/new?labels=type%2Fenhancement&template=feature-request.md)
+ - Raise a proposal to TiKV Repo：[https://github.com/tikv/tikv/issues/new?template=feature-request.md](https://github.com/tikv/tikv/issues/new?template=feature-request.md)
+ - Raise a proposal to PD Repo：[https://github.com/pingcap/pd/issues/new?labels=type%2Fenhancement&template=feature-request.md](https://github.com/pingcap/pd/issues/new?labels=type%2Fenhancement&template=feature-request.md)
+
+**Format requirement**: Add the mark `REQ-PCP` at the beginning of the issue title, for example: 
+
+`REQ-PCP: Further extract tidb_query into different workspaces`
```
Comment (gingerkidney) [inline challenge-programs/challenge-program-season-2.md]:
```suggestion
`REQ-UCP: Further extract tidb_query into different workspaces`
```

**Comment by gingerkidney** [comment]
LGTM

File: challenge-programs/challenge-program-season-2.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
+# Challenge Program Season 2
+
+Welcome to TiDB Challenge Program Season 2.
+
+Season 2 of the TiDB Challenge Program series focuses on the usability of TiDB and its related projects.
+
+Following the principle of “basing directions on users' requirements", we drew most of the requirements of this season from users' votes. These requirements will be presented in the form of tasks in this season. Participants can earn challenge points by claiming tasks. And redeem prizes at the end of the season.
+
+Season 2 of TiDB Challenge Program officially started on March 02, 2020, and will end on May 30, 2020, lasting 3 months.
```
Comment (zz-jason) [inline challenge-programs/challenge-program-season-2.md]:
```suggestion
Season 2 of TiDB Challenge Program officially started on March 02, 2020, and will end on June 02, 2020, lasting 3 months.
```

Comment (zz-jason) [reply challenge-programs/challenge-program-season-2.md]: @gingerkidney Please confirm.

File: challenge-programs/challenge-program-season-2.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
+# Challenge Program Season 2
+
+Welcome to TiDB Challenge Program Season 2.
+
+Season 2 of the TiDB Challenge Program series focuses on the usability of TiDB and its related projects.
+
+Following the principle of “basing directions on users' requirements", we drew most of the requirements of this season from users' votes. These requirements will be presented in the form of tasks in this season. Participants can earn challenge points by claiming tasks. And redeem prizes at the end of the season.
+
+Season 2 of TiDB Challenge Program officially started on March 02, 2020, and will end on June 02, 2020, lasting 3 months.
```
Comment (dcalvin) [inline challenge-programs/challenge-program-season-2.md]:
```suggestion
Season 2 of TiDB Challenge Program officially started on March 02, 2020, and will end on May 30, 2020, lasting 3 months.
```

**Pull request merged**